fableous-fe/src/containers/RegisterPage.tsx

```diff
@@ -0,0 +1,101 @@
+import CardContent from "@material-ui/core/CardContent";
+import Card from "@material-ui/core/Card";
+import { makeStyles } from "@material-ui/core/styles";
+import { Button, TextField, FormControl, Grid } from "@material-ui/core";
+
+import { useState } from "react";
+import { apiClient } from "../Api";
+
+const useStyles = makeStyles({
+  root: {
+    minWidth: 275,
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "center",
+  },
+  bullet: {
+    display: "inline-block",
+    margin: "0 2px",
+    transform: "scale(0.8)",
+  },
+  title: {
+    fontSize: 14,
+  },
+  pos: {
+    marginBottom: 12,
+  },
+});
+
+export default function RegisterPage() {
+  const [userName, setUsername] = useState("");
+  const [eMail, setEmail] = useState("");
+  const [passWord, setPassword] = useState("");
+
+  const postRegister = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    apiClient
+      .post("/api/auth/register", {
```

Comment:
Use restAPI.postRegister from Api.tsx

Reply:
replace string with restApi.auth.postRegister().url

Reply:
Like this instead?
``` ts
axios.request(restAPI.auth.postRegister({ ... }))
```